composer.json

```
@@ -221,6 +221,7 @@
         "phpstan/phpstan-webmozart-assert": "^1.2",
         "phpunit/phpunit": "^9.5",
         "psr/event-dispatcher": "^1.0",
+        "rector/custom-phpspec-to-phpunit": "^0.5.0",
```

```suggestion
        "phpunit/phpunit": "^10.5",
        "psr/event-dispatcher": "^1.0",
        "rector/custom-phpspec-to-phpunit": "dev-main",
```
I made 2 PRs on `rector/custom-phpspec-to-phpunit` repository, to fix some issues I add during the setup of the initial PR I made. Those PRs have been merged already but no tagged version were created yet.

---

Update : v0.6.1 is now released it fixes the missing `setUp` but I think it requires to enable an additional rector rule